crypto/src/test_utils/zero_crypto.rs

```diff
@@ -0,0 +1,400 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * This file contains an implementation of a negligible-cost (apart from some ocassional pk copying) trivial
+ * cryptographic scheme. The purpose of this library is to allow seamless benchmarking of systems without
```

Comment:
Add a WARNING note, that this scheme is 100% unsafe and should never be used in production.